node/src/actor_system_factory.rs

```diff
@@ -1409,6 +1381,7 @@ mod tests {
         );
     }
 
+    #[cfg(feature = "log_recipient_test")]
```

Comment:
This should not be required. Remove it and see what happens. I expect no difference. (For better understanding the reason why look at the line with `make_subject_with_null_setter`; it implicates that setting the global recipient will be just pretence and so no attribute will save it, it's already safe)

Reply:
Done.